deployment/terraform/modules/haystack-apps/kubernetes/pitchfork/main.tf

```diff
@@ -34,7 +33,7 @@ resource "null_resource" "kubectl_apply" {
   provisioner "local-exec" {
    command = "echo '${data.template_file.deployment_yaml.rendered}' | ${var.kubectl_executable_name} apply -f - --context ${var.kubectl_context_name}"
   }
-  count = "${local.count}"
+  count = "${var.pitchfork["enabled"]}"
```

Comment:
wont be it be a problem? as it is boolean not an integer?

Reply:
count in terraform works with boolean